src/main/java/pl/javagame/PasswordManager.java

```diff
@@ -1,35 +1,58 @@
 package pl.javagame;
 
 
-import java.util.ArrayList;
-import java.util.Random;
+import java.util.*;
 
 public class PasswordManager {
 
 
     private ArrayList<String> passwords = new ArrayList<>();
+    private ArrayList<String> passwordsPicked = new ArrayList<>();
+    // private ArrayList<String> copyPasswords = new ArrayList<>();
```

Comment:
Jak kod niepotrzebny to wywalaj, nie ma sensu trzymac zakomentowanego, w razie czego jest w kontroli wersji poprzednia wersja i zawsze mozna wrócic